docs/content/docs/install/kubernetes.md

```diff
@@ -52,7 +71,34 @@ spec:
         pathType: Prefix
 ```
 
-In this example `webhook.host.tld` is the hostname for your pipeline's controller to fill as the webhook URL in the provider platform setup.
+In this example `webhook.host.tld` is the hostname that will be used for the Pipelines as Code webhook URL.
+
+Here's an example with GKE:
+
+```yaml
+apiVersion: networking.k8s.io/v1
```

Comment:
I wonder if we could have some kind of detection of the target platform (ie: gce here) and automatic creation of the Ingress in tkn pac bootstrap the same way we do for openshift and Routes, it would be neat if "tkn pac bootstrap" just works...